File: app/src/main/java/com/example/nijimac103/easyclient_mvvm/view/callback/ProjectClickCallback.java
package com.example.nijimac103.easyclient_mvvm.view.callback;


import com.example.nijimac103.easyclient_mvvm.service.model.Project;

/**
 * クリック操作を伝えるinterface
 * @link onClick(Project project) 詳細画面に移動
 */
public interface ProjectClickCallback {
    void onClick(Project project);
}

File: app/src/main/java/tech/wanderign_engineer/android/githubClient/view/callback/ProjectClickCallback.kt
package tech.wandering_engineer.android.githubClient.view.callback

import tech.wandering_engineer.android.githubClient.service.model.Project

/**
 * クリック操作を伝えるinterface
 * @link onClick(Project project) 詳細画面に移動
 */
interface ProjectClickCallback {
    fun onClick(project: Project)
}
